spec/abstractops.html

```diff
@@ -40,6 +40,20 @@ <h1>ToTimeZoneDisambiguation ( _disambiguation_ )</h1>
     </emu-alg>
   </emu-clause>
 
+  <emu-clause id="sec-temporal-tolargestunit" aoid="ToLargestUnit">
+    <h1>ToLargestUnit ( _largestUnit_, _disallowedUnits_ )</h1>
+    <emu-alg>
+      1. If _largestUnit_ is *undefined*, then
+        1. Return `"days"`.
```

Comment:
This assumption works, but it doesn't sit right with me. I'd rather see either every use provide a non-undefined value (having already checked for undefined and replacing it with a default as is done by Absolute) or for the operation to be more like [ECMA-402 GetOption](http://tc39.es/ecma402/#sec-getoption) in accepting default value as input (e.g., ```ToLargestUnit(_largestUnit_, « `"hours"`, `"minutes"`, `"seconds"` », `"seconds"`)```).

And in anticipation of adding this text to the full ECMA-262, a more specific operation name such as ToLargestTemporalUnit would probably make sense.